CGCharter-1727386911.html

```diff
@@ -62,28 +62,59 @@ <h2 id="goals">
         Incubate new proposals which build on or complement the Social Web WG recommendations.
       </li>
     </ul>
-  
+
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-    <p class="remove">
-      {TBD: Describe topics that are in scope. For specifications that the CLA
-      patent section applies to, it is helpful to describe the scope in a way
-      that it is clear what types of technologies will be defined in
-      specifications, as opposed to adoption by reference or underlying
-      technology not defined in the proposed spec. Key use cases are often
-      helpful in describing scope. If no specifications will be defined in the
-      group that the CLA patent section applies to, the charter should clearly
-      state that. A clear scope is particularly important where patent
-      licensing obligations may apply.}
+    <p>
+      The group will maintain errata and editor&apos;s drafts for the following recommendations from the Social Web Working Group:
+    </p>
+    <ul>
+      <li>ActivityPub</li>
+      <li>Activity Streams 2.0 Core</li>
+      <li>Activity Vocabulary</li>
+      <li>Webmention</li>
+      <li>Linked Data Notifications</li>
+      <li>Micropub</li>
+      <li>WebSub</li>
+    </ul>
+    <p>
+      It will maintain errata and editor&apos;s drafts for the following Notes from the Social Web Working Group:
```

Comment:
```suggestion
      It will maintain errata for the following Notes from the Social Web Working Group:
```

see https://github.com/swicg/potential-charters/pull/22/files#r1833566902